app/assets/javascripts/Components/markus_file_manager.jsx

```diff
@@ -332,7 +332,8 @@ class FileManagerFile extends FileRenderers.RawTableFile {
       name = (
         <a href={this.props.url || "#"} download={this.getName()}>
           {icon}
-          {this.getName()}
+          <span>{this.getName()}</span>
+          {download_icon}
```

Comment:
So in terms of functionality, I want *only* the icon to trigger the download, but not the filename or file icon (`{icon}`). Those parts should be moved outside the `a` tag. But, you can keep the same positioning of the icon to the right of the filename.